### PR TITLE
fix missing types & more improvements for exaroton package ✨

### PR DESCRIPTION
**Pull request by maxiicodes**
added some missing types and more general improvements

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## exaroton ([<kbd>unpkg</kbd>](https://unpkg.com/browse/exaroton@latest/))
was missing the following properties:
1. ServerStatus
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c076eac3255287fe0dd14c6d54e442c8c756073c
</p>

File: types/exaroton/index.d.ts [hunk 1]
```diff
@@ -399,14 +398,14 @@ declare class Server extends EventEmitter {
      *
      * @param streams
      */
-    subscribe(streams?: subscriptionTypes[] | subscriptionTypes): boolean;
+    subscribe(streams?: subscriptionType[] | subscriptionType): boolean;
```
Comment (peterblazejewicz) [inline types/exaroton/index.d.ts:401]: Those are types (a.k.a interfaces). Would be nice, if you could take advantage and change the case (you can retail lowerCase alias for now)

Comment (peterblazejewicz) [reply types/exaroton/index.d.ts:401]: and in other places with `lowerCamelCase` notation to `UpperCamelCase`

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by hofmmaxi** [comment]
Ready to merge

**Pull request merged**